src/Sarif.Viewer.VisualStudio/Tags/SarifLocationTagHelpers.cs

```diff
@@ -1,25 +1,30 @@
-﻿// Copyright (c) Microsoft. All rights reserved. 
-// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.VisualStudio.ComponentModelHost;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Text;
 
 namespace Microsoft.Sarif.Viewer.Tags
 {
-    using Microsoft.VisualStudio.ComponentModelHost;
-    using Microsoft.VisualStudio.Shell;
-
     internal static class SarifLocationTagHelpers
     {
         /// <summary>
         /// Calls into the tagger service and asks it to refresh the tags being displayed in Visual Studio.
         /// </summary>
-        public static void RefreshAllTags()
+        /// <param name="textBuffer">
+        /// The text buffer whose tags are to be refeshed, or null if the tags for all text buffers
+        /// are to be refreshed.
+        /// </param>
+        public static void RefreshAllTags(ITextBuffer textBuffer = null)
```

Comment:
>RefreshAllTags [](start = 27, length = 14)

Have to think about this a bit... RefreshAllTags to me means, refresh everything...

Having it take a parameter (in my simple mind) seems odd.

That being said, I'm not that concerned about it

Reply:
Per discussion, merging without change.

---
In reply to: [516218203](https://github.com/microsoft/sarif-visualstudio-extension/pull/245#discussion_r516218203) [](ancestors = 516218203)

Reply:
... aaaand, you're right, I'll change it in a separate PR.

---
In reply to: [516219754](https://github.com/microsoft/sarif-visualstudio-extension/pull/245#discussion_r516219754) [](ancestors = 516219754,516218203)